packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -323,24 +323,20 @@ export class FeatureDevController {
 
         getLogger().info(`${featureName} conversation id: ${session.conversationId}`)
 
+        // This is a loading animation
         this.messenger.sendAnswer({
-            type: 'answer',
+            type: 'answer-stream',
             tabID,
             message: approachCreation,
         })
 
-        // Ensure that the loading icon stays showing
-        this.messenger.sendAsyncEventProgress(tabID, true, undefined)
-
-        this.messenger.sendUpdatePlaceholder(tabID, 'Generating plan ...')
```

Comment:
Since the plan is not streaming but comes in a single chunk (and soon it will be snap to top of the page instead of scrolling to bottom) we're currently using the original card which indicates `Ok, let me create a plan` for the animation card. Instead of creating empty cards, it is both better in UX wise and also the handling of the cards.

Reply:
Thats makes sense but the `"Generating plan ..."` is just the placeholder input text isn't it?

Reply:
It was causing the remain as a streaming card and even though we had the plan it was still spinning as an animation. Now the flow is like that:
`Ok, let me create a plan` card appears and holds on the screen with the bottom border animation.
When the plan is ready, a new card appears with the plan details directly and the `let me create` card animation stops.
`PLAN_PLAN_PLAN` (no animation or sth.)
The rest is same.

Reply:
I think we're talking about two separate things maybe. 

When a user initiates the plan generation flow it the bottom input box should be locked and say "Generating plan ..." (its separate from whats showing in any of the cards)

Reply:
Is it possible for updating the text like this to mess with the cards? I would have thought the systems were seperate?

This is an example of what updating the placeholder would do (stolen from the image in the pr description)
<img width="694" alt="Screenshot 2024-05-30 at 9 03 57 AM" src="https://github.com/aws/aws-toolkit-vscode/assets/103940141/a8a08c92-75de-4fe6-a0ae-e16bae080ca8">

Reply:
Yeah yeah you're 100% right. I've missed the placeholder indication. I was talking about the previous one during the whole discussion:
`this.messenger.sendAsyncEventProgress(tabID, true, undefined)` this is not relevant anymore however placeholder should remain same